concent_api/core/tests/test_payments_bankster.py

```diff
@@ -360,6 +360,35 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount
 
         self.assertEqual(claim_against_requestor.amount, 1000)
 
+    def test_that_settle_overdue_acceptances_should_raise_exception_if_any_transaction_from_acceptances_list_is_before_youngest_transaction_timestamp(self):
```

Comment:
how this scenario is different from `test_that_settle_overdue_acceptances_should_return_none_if_subtask_costs_where_already_paid`? I thought that if any of transactions from `SRA` list is before the youngest transaction timestamp, this is an indicator that subtask costs have been already paid.

Reply:
The effect is different (return None vs raising exception) so it tests the different flow.

> I thought that if any of transactions from SRA list is before the youngest transaction timestamp, this is an indicator that subtask costs have been already paid.

Not necessary, client may try to manipulate the list to ie. try to get payment for already payed transaction.